test/__snapshots__/mathml-spec.js.snap

```diff
@@ -209,32 +209,30 @@ exports[`A MathML builder should render boldsymbol with the correct mathvariants
 <math>
   <semantics>
     <mrow>
-      <mrow>
```

Comment:
Not sure what cause this change, but I'm happy to see it.

Reply:
I added a change to remove the outer mrow if all it contained was an mrow or mtable. This is especially nice (I think) for tagged equations, but thought I'd add it for mrow too.